### PR TITLE
Add some Rubyism

### PR DESCRIPTION
**Pull request by Daniel-N-Huss**
Hey! 

Leaving this as a draft, as I'm not sure this set of suggestions is in a good enough state as is, but I wanted to see if there was some good discussion to be had.

As I was watching the demo video discussing Time sorting, I had some flashbacks to the number of mistakes I've made trying to compare times: `.before?` and `.after?` are helpful from the Rails side, but I wonder if there's room for aliasing some of these methods to get some expressiveness in here.

I had some fun poking at Oldest in particular. It's a tough word to sort with; Measuring someones age is a different meaning than the oldest time someone cleaned the breakroom.

Part of my hesitation with a full PR is that I know I'm overlooking alternative uses of these aliases that would behave in an unexpected way, and undermine the core value of what's here.

I also am not sure what a good place to put these aliases are! I considered breaking them out to another module, to leave the main API very focused, and thin. But, aliasing class methods from inside an included module didn't seem right. If aliasing methods like `older` got brought in, is there a better place to keep them?

Final thought: I debated some sort of `Delegate`, where `oldest` can delegate to the core `asc / desc` methods, but my raw ruby knowledge is lacking: is there a better way to pass along messages to different methods on the class? Or am I thinking about this wrong, and there's a better pattern altogether?

### CODE REVIEW

**Comment by Daniel-N-Huss** [comment]
Reconsidering this suggestion, alongside some feedback. 

For posterity, this is opening up too many ambiguous interpretations to effectively maintain. While Ruby has things like `first` and `second`, those are incredibly clear and really only have one way of being implemented. 

So much of what I’ve suggested will ultimately fall on context-dependent aliasing, that takes away from the core value prop here. Domain will inevitably leak into the gem by opening this pathway up, and enable unexpected results. 


Perhaps I’ll poke at something config-like, that would enable self-defined aliasing, if that seems valuable while utilizing! 

TLDR: thanks for putting this cool gem together, and humouring a wayward PR!
